### ISSUE TITLE
Login form has a <p> with no text in it. It's the <p> for the Submit Button

### ISSUE THREAD

**Issue by mathetos** [WordImpress/Give#332]
## A11Y Report Notes
http://b8d.3e8.myftpupload.com/give-shortcode-profile/
**Give Shortcode: Profile**
* Error /  priority 86%
* Empty paragraph found
* (line: 127, 4)

```
<p>
	<input type="hidden" name="give_redirect" value="http://b8d.3e8.myftpupload.com/give-shortcode-profile/" />
	<input type="hidden" name="give_login_nonce" value="1c89f72793" />
	<input type="hidden" name="give_action" value="user_login" />
	<input id="give_login_submit" type="submit" class="give_submit" value="Log In" />
</p>
```

**Web Content Accessibility Guidelines (WCAG) 2.0, Level AA: 1.4.5 Images of Text**
A paragraph was found without text in it. While this tends to have a low severity, it does create an unnecessary nuisance for users of screen readers who are likely to hear the announcement of the empty paragraph as they navigate through content. If this empty paragraph exists to behave as margin, use CSS for this purpose instead.

## Possible Solutions
* Hidden text in a `<span>` Element? Feels hacky.
* Output the shortcode without `<p>` breaks alltogether, most likely happening from wpautop function. 
* Output this markup more symantically in general rather than with p's use Div's or LI's

**Comment by TheGeniusWM** [comment]
Recommended to replace the <p> with some css <div> that serves the same purpose.
Stick to using the <p> for text only, not chunks of code.
Should be an easy fix?

**Comment by mathetos** [comment]
Yep, working on that now.
